percona-mongo/1.10/README.md

```diff
@@ -0,0 +1,652 @@
+
+## General limitations
+
+Below are some general limitations of the service.
+
+
+## Service user		
+		
+The DC/OS Mongo Service uses a Docker image to manage its dependencies for Percona Server for MongoDB. Since the Docker image contains a full Linux userspace with its own `/etc/users` file, it is possible for the default service user `nobody` to have a different UID inside the container than on the host system. Although user `nobody` has UID `65534` by convention on many systems, this is not always the case. As Mesos does not perform UID mapping between Linux user namespaces, specifying a service user of `nobody` in this case will cause access failures when the container user attempts to open or execute a filesystem resource owned by a user with a different UID, preventing the service from launching. If the hosts in your cluster have a UID for `nobody` other than 65534, you will need to specify a service user of `root` to run DC/OS Mongo Service.
+		
+To determine the UID of `nobody`, run the command `id nobody` on a host in your cluster:		
+```		
+$ id nobody		
+uid=65534(nobody) gid=65534(nobody) groups=65534(nobody)		
+```		
+		
+If the returned UID is not `65534`, then the DC/OS Mongo Service can be installed as root by setting the service user at install time:		
+```		
+"service": {		
+        "user": "root",		
+        ...		
+}		
+...		
+```
+
+<a name="removing-a-node"></a>
+## Removing a Node
+
+Removing a node is not supported at this time.
+
+<a name="only-root-volumes"></a>
+## Only ROOT Volumes Supported
+
+The service only supports ROOT volumes and not MOUNT volumes with absolute paths because of a validation error. This requires a custom scheduler.
+
+<a name="updating-storage-volumes"></a>
+## Updating Storage Volumes
+
+Volume size requirements may be changed after initial deployment.
+
+<a name="rack-aware-replication"></a>
+## Rack-aware Replication
+
+Rack placement and awareness are not supported at this time.
+
+## Overlay network configuration updates
+When a pod from your service uses the overlay network, it does not use the port resources on the agent machine, and thus does not have them reserved. For this reason, we do not allow a pod deployed on the overlay network to be updated (moved) to the host network, because we cannot guarantee that the machine with the reserved volumes will have ports available. To make the reasoning simpler, we also do not allow for pods to be moved from the host network to the overlay. Once you pick a networking paradigm for your service the service is bound to that networking paradigm.
+
+<a name="support"></a>
+# Support
+
+<a name="package-versioning-scheme"></a>
+## Package Versioning Scheme
+
+- MongoDB: See the framework description
+- DC/OS: 1.10
+
+<a name="contacting-technical-support"></a>
+## Contacting Technical Support
```

Comment:
Change this section to just say `Contact mesosphere@percona.com` for now. We'll update it with support contacts with beta release.

Reply:
👍